scripts/loadtest-tools.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
this script is just so we can spin up a utility dyno and quickly run a load test. run it with e.g. `heroku run -s performance-m -a ft-google-amp-memory-test /app/scripts/loadtest-tools.sh`